src/lib/util/useSWRWithErrorToast.tsx

```diff
@@ -34,7 +34,8 @@ export default function useSWRWithErrorToast<Data, ErrorType>(
   transform: ErrorTransformerProps<ErrorType>,
```

Comment:
transform is a pretty confusing name for an error renderer

Reply:
True!